src/main/java/org/folio/circulation/services/PubSubPublishingService.java

```diff
@@ -58,54 +58,15 @@ public PubSubPublishingService(WebContext context) {
 
           if (throwable != null && throwable.getMessage() != null &&
             throwable.getMessage().toLowerCase().contains(
-            "there is no subscribers registered for event type")) {
+              "there is no subscribers registered for event type")) {
 
             publishResult.complete(true);
-          }
-          else {
+          } else {
             publishResult.completeExceptionally(throwable);
           }
         }
       });
 
     return publishResult;
   }
-
-  public PubsubClient getPubSubClient() {
-    return pubSubClient;
-  }
-
-  /**
-   * Method that publishes event to PubSub and includes response body in the exception message in
-   * case of any response status other than 204.
-   *
-   * @param eventMessage Event to be published
-   * @return
-   */
-  private CompletableFuture<Boolean> sendEventMessage(Event eventMessage) {
-    CompletableFuture<Boolean> result = new CompletableFuture<>();
-
-    try {
-      getPubSubClient().postPubsubPublish(eventMessage, ar -> {
-        if (ar.statusCode() == HttpStatus.HTTP_NO_CONTENT.toInt()) {
-          result.complete(true);
-        } else {
-          ar.bodyHandler(body -> {
-            EventSendingException exception = new EventSendingException(String.format(
-            "Error during publishing Event Message in PubSub. " +
-              "Status code: %s . Status message: %s . Response body: %s",
-            ar.statusCode(), ar.statusMessage(), body.toString()));
-            logger.error(exception);
-            result.completeExceptionally(exception);
-          });
-        }
-      });
-      return result;
-    } catch (Exception ex) {
-      logger.error("Error during sending event message to PubSub, event {}", ex,
-        eventMessage.getEventPayload());
-      result.completeExceptionally(ex);
-      return result;
-    }
-  }
```

Comment:
This logic is in the method above, as far as I see.

Reply:
> This logic is in the method above, as far as I see.

Which method are you referring to? The diff I'm viewing does not have a method above that has changes?

Reply:
This one `public CompletableFuture<Boolean> publishEvent(String eventType, String payload)`

Reply:
@marcjohnson-kint, this method duplicates the PubsubClientUtil method, events are not rejected if there are no subscribers since v1.2.0

Reply:
> This one `public CompletableFuture<Boolean> publishEvent(String eventType, String payload)`

Is that because it now uses PubSubClientUtils?

> this method duplicates the PubsubClientUtil method, events are not rejected if there are no subscribers since v1.2.0

Assuming the util methods are used and not the actual client?